service/controller/resource/blobobject/desired.go

```diff
@@ -63,7 +49,7 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 
 	var ignitionTemplateData cloudconfig.IgnitionTemplateData
 	{
-		versions, err := v_6_0_0.ExtractComponentVersions(release.Spec.Components)
+		versions, err := v_6_0_0.ExtractComponentVersions(cc.Release.Components)
```

Comment:
why did we move it to the controller context?

Reply:
@MarcelMue did this, would you like to answer?

Reply:
It was nicer for me to have a dedicated resource to get the Releases once and then take them from the context when I needed them.

Just personal preference I guess.